concrete/src/Entity/Express/Entry.php

```diff
@@ -382,7 +382,6 @@ public function __construct()
     {
         $this->attributes = new ArrayCollection();
         $this->associations = new ArrayCollection();
-        $this->containing_associations = new ArrayCollection();
```

Comment:
This `containing_associations` shoudn't be there, right? I couldn't find any reference to it in the core.